libs/sync/sync.go

```diff
@@ -3,18 +3,27 @@ package sync
 import (
 	"context"
 	"fmt"
+	"path/filepath"
+	"slices"
 	"time"
 
 	"github.com/databricks/cli/libs/filer"
+	"github.com/databricks/cli/libs/fileset"
 	"github.com/databricks/cli/libs/git"
 	"github.com/databricks/cli/libs/log"
 	"github.com/databricks/databricks-sdk-go"
 	"github.com/databricks/databricks-sdk-go/service/iam"
 )
 
+// This directory is used to store and automaticaly sync internal bundle files, such as, f.e
+// notebook trampoline files for Python wheel and etc.
+const internalFolder = ".internal"
+
```

Comment:
This should be removed from `libs/sync`. The bundle code should append the path to the list of includes.